solr/core/src/java/org/apache/solr/security/JWTPrincipalWithUserRoles.java

```diff
@@ -61,11 +58,17 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "JWTPrincipalWithUserRoles{" +
-        "username='" + username + '\'' +
-        ", token='" + "*****" + '\'' +
-        ", claims=" + claims +
-        ", roles=" + roles +
-        '}';
+    return "JWTPrincipalWithUserRoles{"
+        + "username='"
+        + username
+        + '\''
```

Comment:
this has become more awkward. It could be improved by combining the apostrophe into the string of the next line; same for further below.

Reply:
Used String.format() instead